src/Basque/Mews.Fiscalizations.Basque/TicketBaiClient.cs

```diff
@@ -123,15 +123,30 @@ private async Task<SendInvoiceResponse> SendTicketBaiInvoiceAsync(TicketBaiInvoi
         var response = await HttpClient.PostAsync(ServiceInfo.SendInvoiceUri(Environment), requestContent);
 
         var responseContent = await response.Content.ReadAsStringAsync();
-        var ticketBaiResponse = XmlSerializer.Deserialize<Dto.TicketBaiResponse>(responseContent);
-        return DtoToModelConverter.Convert(
-            response: ticketBaiResponse,
-            qrCodeUri: invoiceData.QrCodeUri,
-            xmlRequestContent: signedRequest.OuterXml,
-            xmlResponseContent: responseContent,
-            tbaiIdentifier: invoiceData.TbaiIdentifier,
-            signatureValue: invoiceData.TrimmedSignature
-        );
+
+        try
+        {
+            var ticketBaiResponse = XmlSerializer.Deserialize<TicketBaiResponse>(responseContent);
```

Comment:
Did we check the technical documentation to see if they return a specific response format in case of a server error?

I think its best if we can do something like this but of course that depends if they return some specific format in case of a server error:
if (success) -> deserialize the response to TicketBaiResponse
if (error) -> deserialize the error response
if something is unhandled -> return some custom error response.